File: AtCoder/ABC177/ABC177A.cs
using System;
using System.Linq;

public class ABC {
    private static void Main(string[] args) {
        var input = ReadDoublesAsArray();
        var d = input[0];
        var t = input[1];
        var s = input[2];

        var answer = (d / s <= t) ? "Yes" : "No";
        Console.WriteLine(answer);
    }

    private static double[] ReadDoublesAsArray() => Console.ReadLine().Trim().Split().Select(x => double.Parse(x)).ToArray();
}

File: AtCoder/ABC177/ABC177A.kt
fun main(args: Array<String>) {
    val input = readLine()!!.split(" ").map(String::toDouble)
    val d = input[0]
    val t = input[1]
    val s = input[2]

    val answer = if (d / s <= t) "Yes" else "No"
    println(answer)
}